CMakeLists.txt

```diff
@@ -0,0 +1,49 @@
+cmake_minimum_required(VERSION 3.18...3.26 FATAL_ERROR)
+
+project(DynaTest LANGUAGES CXX
+        VERSION 3.0)
+
+list(APPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/TangoCMakeModules)
```

Comment:
The TangoCMakeModules git submodule seems to be missing at first sight too.

Reply:
Well for cppTango 10.0.0 we don't need the tango cmake modules anymore. Should I add them for older versions though?

Reply:
Okay, I've removed that line.